spec/rubocop/cop/style/if_unless_modifier_spec.rb

```diff
@@ -421,6 +421,26 @@ def f
     expect(corrected).to eq "a = (1 if b)\n"
   end
 
+  context 'when the code before condition pushes inline length over limit' do
+    let(:line_length_config) { { 'Enabled' => true, 'Max' => 100 } }
+
+    it 'does not result in infinite loop with assignment' do
+      expect_no_offenses(<<~RUBY)
+        timespan = if params[:timespan] && POSSIBLE_TIMESPANS.include?(params[:timespan])
+                     params[:timespan].to_sym
+                   end
+      RUBY
+    end
+
+    it 'does not result in infinite loop with any other statement' do
```

Comment:
An infinite loop error is caused by a combination of cops. Con you add a reproduction test for infinite loop error to spec/rubocop/cli/cli_autocorrect_spec.rb? E.g.: https://github.com/rubocop-hq/rubocop/pull/7901/files#diff-0e7dbdd6e99e0defc2bab2cb544b42da
And I think the description of this test better to be updated to not register an offense.